### ISSUE TITLE
[suggestion] add coordinates to text file

### ISSUE THREAD

**Issue by bomba1749** [maruohon/tweakeroo#185]
it would be really nice to have a tweak that adds your coordinates to a text file, upon a hotkey press. This would be really useful for being able to mark locations of resources. Ideally, it would work like this: a key bound to it is pressed; then a text input box would show up. Then, once the player put their message in the box, the game would save their coordinates to a text file, like this example:

~~~~~~~~~~~~~
1289 64 -8213
Lots of clay here
~~~~~~~~~~~~~

**Comment by maruohon** [comment]
I can add this... but isn't this something you usually use the waypoints in a minimap mod for?

**Comment by bomba1749** [comment]
yeah, but it's also useful to be able to have them in a list in a text file so you can quickly sort and compare them, or if someone else needs a bunch of coordinates you can just send the file
